tests/core/test_parsing.py

```diff
@@ -234,60 +237,63 @@ def test_parse_notes(page, start, end):
     assert notes.endswith(end)
 
 
-@pytest.mark.parametrize('page,start,end', (
-    ('traffic-fatality-2-3', 'The preliminary investigation shows that the grey',
-     'No charges are expected to be filed.'),
-    ('traffic-fatality-4-6', 'The preliminary investigation shows that a black, Ford', 'scene at 01:48 a.m.'),
-    ('traffic-fatality-15-4', 'The preliminary investigation indicated that Garrett',
-     'seatbelts. No charges are expected to be filed.'),
-    ('traffic-fatality-16-4', 'The preliminary investigation revealed that the 2017', 'injuries on April 4, 2019.'),
-    ('traffic-fatality-17-4', 'The preliminary investigation revealed that the 2010', 'at the time of the crash.'),
-    ('traffic-fatality-20-4', 'The preliminary investigation revealed that a 2016',
-     'pronounced deceased at the scene.'),
-    ('traffic-fatality-25-4', 'Suspect Vehicle:  dark colored', 'damage to its right, front end.'),
-    ('traffic-fatality-71-2', 'The preliminary investigation shows that a 2004 Honda sedan',
-     'at the scene at 8:50 p.m.'),
-    ('traffic-fatality-72-1', 'The preliminary investigation shows that the 2016 Indian',
-     'whether charges will be filed.'),
-    ('traffic-fatality-73-2', 'The preliminary investigation shows that a 2003 Ford F150',
-     'St. David’s South Austin Hospital.'),
-))
-def test_parse_notes_field(page, start, end):
-    page_text = load_test_page(page)
-    parsed_content, r = parsing.parse_page_content(page_text)
-    notes = parsed_content[Fields.NOTES]
+@pytest.mark.parametrize(
+    'raw_notes,split_word,start,end',
+    [
+        pytest.param(
+            mock_deceased.DECEASED_WITH_NOTES_00, '02/15/1960', 'The preliminary', 'be filed.', id='big-paragragh'),
+        pytest.param(
+            mock_deceased.DECEASED_WITH_NOTES_01, '12/31/1960', 'The preliminary', '01:48 a.m.', id='split-paragraphs'),
+        pytest.param(mock_deceased.DECEASED_WITH_NOTES_02,
+                     '11/13/1991',
+                     'The preliminary',
+                     'be filed.',
+                     id='multi-dedeceased-one-paragraphs'),
+        pytest.param(mock_deceased.DECEASED_WITH_NOTES_03,
+                     '01/26/1992',
+                     'The preliminary',
+                     'contacting them.',
+                     id='multi-deceased-fields'),
+    ],
+)
+def test_parse_notes_field(raw_notes, split_word, start, end):
+    """Ensure notes are ."""
+    note_soup = parsing.to_soup(raw_notes)
+    notes = parsing.parse_notes_field(note_soup, split_word)
     assert notes.startswith(start)
     assert notes.endswith(end)
 
 
-def test_no_DOB_field_when_DOB_not_provided():
```

Comment:
Why delete this test? Is it covered somewhere else?

Reply:
Yes, it was a duplicate of this use case: https://github.com/rgreinho/scrapd/blob/remove-test-pages/tests/core/test_person.py#L71-L78

Reply:
I don't think it is covered. The test you kept just checks that the needed fields are correct, not that there's no unneeded field. I needed to add `test_no_DOB_field_when_DOB_not_provided` to catch a bug I was seeing when I ran the CLI.

I'm sure it would be feasible to consolidate them into one test that covered both issues, but it's not there yet.

Reply:
Ah I see! Good point. That's sort of the negative of the test I pointed you too then. I'll readd it.

Also, with the models we won't need to to that 🎉

Reply:
Actually we can keep it deleted, as this the next series of test I am working on will test that the output looks exactly how it is expected to look. Therefore the negative tests will be covered for free.